CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-UNRELEASED
```

Comment:
AFAIK this change is necessary. At least in the past it worked for me to not change it and then it was done automatically. I don't know however if this breaks anything :thinking:

Reply:
You're right! 
`/lunes-cms/.circleci/config.yml:246`